Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/06-SyncUpDetail/EditingAndDeletingSyncUp.tutorial

```diff
@@ -184,7 +184,8 @@
       }
       
       @Step {
-        Use the ``ComposableArchitecture/Reducer/ifLet(_:action:then:fileID:line:)-42kki`` operator
+        Add support for the `.alert` case, and 
```

Comment:
because the step also does this in the diff from
https://github.com/pointfreeco/swift-composable-architecture/blob/af38644542b1310c0bb023a5af2545beb8854009/Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/06-SyncUpDetail/EditingAndDeletingSyncUp-02-code-0003.swift#L28-L29
to
https://github.com/pointfreeco/swift-composable-architecture/blob/af38644542b1310c0bb023a5af2545beb8854009/Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/06-SyncUpDetail/EditingAndDeletingSyncUp-02-code-0004.swift#L28-L32